### DISCUSSION TITLE
Support specifying helmfile/helmfile.d using environment variable

### DISCUSSION THREAD

**Discussion by sstarcher** [helmfile/helmfile#1254] [Ideas]
An example is a repo where users want to be able to run helmfile anywhere in the directory, but they don't want to also have to specify the file.  Such as `helmfile -f $REPO_HOME/gitops/helmfile.d list` instead an environment variable could be set `HELMFILE_FILE_PATH=$REPO_HOME/gitops/helmfile.d` and while in the repo the user can now run `helmfile list` without worrying about the path.

Another option could be doing a recursive upward directory search like git does to find a helmfile or helmfile.d

**Comment by yxxhero** [comment]
@sstarcher PR is welcome.

**Reply by sstarcher** [reply]
Any preference from a functionality standpoint between the two?  Or both?

**Reply by sstarcher** [reply]
https://github.com/helmfile/helmfile/pull/1255
